akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/BehaviorTestKit.scala

```diff
@@ -4,24 +4,39 @@
 
 package akka.actor.testkit.typed.javadsl
 
-import java.util.concurrent.ThreadLocalRandom
-
+import akka.actor.testkit.typed.internal.{ ActorSystemStub, BehaviorTestKitImpl }
 import akka.actor.testkit.typed.{ CapturedLogEvent, Effect }
-import akka.actor.testkit.typed.internal.BehaviorTestKitImpl
-import akka.actor.typed.{ ActorRef, Behavior, Signal }
 import akka.actor.typed.receptionist.Receptionist
+import akka.actor.typed.{ ActorRef, Behavior, Signal }
 import akka.annotation.{ ApiMayChange, DoNotInherit }
+import com.typesafe.config.Config
+
+import java.util.concurrent.ThreadLocalRandom
 
 object BehaviorTestKit {
-  import akka.actor.testkit.typed.scaladsl.TestInbox.address
 
   /**
    * JAVA API
    */
   @ApiMayChange
-  def create[T](initialBehavior: Behavior[T], name: String): BehaviorTestKit[T] = {
+  def ApplicationTestConfig: Config = akka.actor.testkit.typed.scaladsl.BehaviorTestKit.ApplicationTestConfig
```

Comment:
That is not normal Java naming convention.
Either it should be shown as a constant:
```
val APPLICATION_TEST_CONFIG: Config
```

or as an ordinary method:
```
def applicationTestConfig(): Config
```

I think I vote for the latter.

Reply:
this is simply forwarding to the already existing val in `akka.actor.testkit.typed.scaladsl.BehaviorTestKit`, I figured that if the naming was right there it's right here as well. I preferred `def` for forwarding but I have no objecting changing this to a val.

Reply:
this is now changed to match the `ActorTestKit` Java API